Divine/CLI/CommandLineActions.cs

```diff
@@ -0,0 +1,150 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Divine.Enums;
+using LSLib.LS;
+
+namespace Divine.CLI
+{
+    internal class CommandLineActions
+    {
+        public static string SourcePath;
+        public static string DestinationPath;
+        public static string ConformPath;
+
+        public static Game Game;
+        public static LogLevel LogLevel;
+        public static ResourceFormat InputFormat;
+        public static ResourceFormat OutputFormat;
+        public static PackageVersion PackageVersion;
+        public static Dictionary<string, bool> GraphicsOptions;
+
+        // TODO: OSI support
+
+        public static void Run(CommandLineArguments args)
+        {
+            SetUpAndValidate(args);
+            Process(args);
+        }
+
+        private static void SetUpAndValidate(CommandLineArguments args)
+        {
+            string[] batchActions = { "extract-packages", "convert-models", "convert-resources" };
+            string[] packageActions = { "create-package", "extract-package", "extract-packages" };
+            string[] graphicsActions = { "convert-model", "convert-models" };
+
+            LogLevel = CommandLineArguments.GetLogLevelByString(args.LogLevel);
+            CommandLineLogger.LogInfo($"Using log level: {LogLevel}");

```

Comment:
The "INFO" (Informational) log level should probably be reserved for messages that are actually useful for the end-user. I 'd say the log level should be a DEBUG message.